src/utils/componentService.js

```diff
@@ -0,0 +1,28 @@
+import api from '../config/api'
+
+const createQuery = (target = '', query) => `
+query {
+  repository(owner: "Wolox", name: "frontend-cookbook") {
```

Comment:
This should go in `.env` file or isn't necessary?

Reply:
Not really,  this can be in here since it's not sensible data (at least not for now) 😄